python/cuml/linear_model/base.pyx

```diff
@@ -66,6 +68,21 @@ class LinearPredictMixin:
         Predicts `y` values for `X`.
 
         """
+        coef_cp, n_feat, n_targets, _ = input_to_cupy_array(self.coef_)
```

Comment:
Looks like `n_feat` is not used. Also, would it be possible to directly inspect `self.coef_` so that the full conversion can be delayed into the `1 < n_targets` branch?

Reply:
Yes. Fixed.